docs/api/members.rst

```diff
@@ -34,6 +34,14 @@ RawGuildMemberRemoveEvent
 .. autoclass:: RawGuildMemberRemoveEvent()
     :members:
 
+RawPresenceUpdateEvent
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. attributetable:: RawGuildMemberRemoveEvent
```

Comment:
copy-paste oversight? (been there, done that)

Reply:
6740164b oops